lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8452,14 +8476,10 @@ bool SwiftASTContext::GetCompileUnitImportsImpl(
   if (cu_imports.size() == 0)
     return true;
 
-  Progress progress(
-      llvm::formatv("Getting Swift compile unit imports for '{0}'",
-                    compile_unit->GetPrimaryFile().GetFilename()),
-      cu_imports.size());
-
+  Progress progress("Getting Swift compile unit imports");
```

Comment:
Suggestion: "Importing modules used by compilation unit {0}", compile_unit->GetPrimaryFile().GetFilename()

Reply:
Good suggestion, I'll change this.